server/weaverbird/backends/sql_translator/steps/fromdate.py

```diff
@@ -0,0 +1,57 @@
+from distutils import log
+
+from weaverbird.backends.sql_translator.steps.utils.query_transformation import (
+    build_selection_query,
+    complete_fields,
+    snowflake_date_format,
+)
+from weaverbird.backends.sql_translator.types import (
+    SQLPipelineTranslator,
+    SQLQuery,
+    SQLQueryDescriber,
+    SQLQueryRetriever,
+)
+from weaverbird.pipeline.steps import FromdateStep
+
+
+def translate_fromdate(
+    step: FromdateStep,
+    query: SQLQuery,
+    index: int,
+    sql_query_retriever: SQLQueryRetriever = None,
+    sql_query_describer: SQLQueryDescriber = None,
+    sql_translate_pipeline: SQLPipelineTranslator = None,
+) -> SQLQuery:
+    query_name = f'FROMDATE_STEP_{index}'
+
+    log.debug(
+        "############################################################"
+        f"query_name: {query_name}\n"
+        "------------------------------------------------------------"
+        f"step.name: {step.name}\n"
+        f"step.column: {step.column}\n"
+        f"step.format: {step.format}\n"
+        f"query.transformed_query: {query.transformed_query}\n"
+        f"query.metadata_manager.tables_metadata: {query.metadata_manager.tables_metadata}\n"
+        f"query.metadata_manager.query_metadata: {query.metadata_manager.query_metadata}\n"
+    )
+    step.format = snowflake_date_format(step.format)
+    # we change the column type
+    query.metadata_manager.change_type(step.column, 'text')
```

Comment:
```suggestion
    query.metadata_manager.change_type(step.column, 'str')
```
La liste des types que l'on utilise est dispo ici: https://github.com/ToucanToco/toucan-connectors/blob/9ee0f48448e17b67b7def559e994ff65148b5b4c/toucan_connectors/snowflake_common.py#L14

Reply:
updated !